extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetTableWriter.java

```diff
@@ -952,14 +953,16 @@ public int bufferFetchedData() {
                 return 0;
             }
             buffer.clear();
-            for (int i = 0; i < chunk.size(); i++) {
-                buffer.put(chunk.get(i));
+            for (int chunkIdx = 0; chunkIdx < chunk.size(); chunkIdx++) {
```

Comment:
I think you'll get better performance if you make this a bulk transfer function.  Instead of making `chunk.size()` virtual calls (with the virtualization overhead), you can make a single call and pass in the size.